docs/how_to/render_plane_manipulator.md

```diff
@@ -19,7 +19,7 @@ the rendering plane.
      napari
      ```
 
-2. Load the "HIV virus-like particles tomogram" image from the File menu: File -> Open Sample -> HIV virus-like particle tomogram. Note that this will download the sample image so it make take some time. If you have previously opened the sample image, it should be cached and will not download again.
+2. Load the "HIV virus-like particles tomogram" image from the File menu: **File -> Open Sample -> HIV virus-like particles tomogram**. Note that this will download the [sample image from Zenodo](https://doi.org/10.5281/zenodo.6504891) (~470 Mb) so it make take some time. If you have previously opened the sample image, it should be cached and will not download again.
```

Comment:
```suggestion
2. Load the "HIV virus-like particles tomogram" image from the File menu: **File -> Open Sample -> HIV virus-like particle tomogram**. Note that this will download the [sample image from Zenodo](https://doi.org/10.5281/zenodo.6504891) (~470 Mb) so it make take some time. If you have previously opened the sample image, it should be cached and will not download again.
```

Typo, it should be "particle" not "particles", to match what is actually shown in the menu

Reply:
Extra suggestion: I think people will avoid clicking the zenodo link because the text here implies it will start an automatic download. Can we re-phrase to make it more obvious the link is to the data description page instead?

Eg:
...this will download the sample image from Zenodo (~470 Mb), for more details see the [data description page here](https://doi.org/10.5281/zenodo.6504891).